Documentation/CatalogManagement/Initialization-File.md

```diff
@@ -59,7 +61,14 @@ Catalog files can be edited three ways:
 
 ## Catalog file properties
 
-### Setting camera positions
+### `corsDomains`
+
+By default, Terria proxies all requests, assuming that the servers do not support CORS. You can add hosts that are known to support CORS to this property to avoid proxying them. (This property is also supported by TerriaJS-Server).
```

Comment:
Not quite accurate.  It proxies all requests that are in the proxy whitelist, not "all requests".  And there's no corresponding property on `terriajs-server` as far as I know.